tests/acceptance/features/apiGraph/unassignRole.feature

```diff
@@ -24,7 +24,7 @@ Feature: unassign user role
       | User        |
       | Guest       |
 
-  @issue-6035
```

Comment:
I think it's okay to leave the issue tag even if the issue has been solved, for documentation purpose

Reply:
IMO, it will be less confusing if we remove the fixed issues tag.

Reply:
It can help in the future if the scenario fails. Having an issue tag will be helpful to go through the previous issues the scenario encountered